docker/Dockerfile

```diff
@@ -34,12 +34,10 @@ RUN groupadd -g $ARG_GID anythingllm && \
 # Copy docker helper scripts
 COPY ./docker/docker-entrypoint.sh /usr/local/bin/
 COPY ./docker/docker-healthcheck.sh /usr/local/bin/
-COPY ./docker/dual_boot.sh /usr/local/bin/
 
 # Ensure the scripts are executable
 RUN chmod +x /usr/local/bin/docker-entrypoint.sh && \
-    chmod +x /usr/local/bin/docker-healthcheck.sh && \
-    chmod 777 /usr/local/bin/dual_boot.sh
```

Comment:
We cant remove these - they are how the docker image dual boots both the anythingllm app & the python document processor API!